packages/flutter/lib/src/widgets/animated_cross_fade.dart

```diff
@@ -151,15 +151,23 @@ class _AnimatedCrossFadeState extends State<AnimatedCrossFade> with TickerProvid
   }
 
   Animation<double> _initAnimation(Curve curve, bool inverted) {
-    final CurvedAnimation animation = new CurvedAnimation(
+    Animation<double> animation = new CurvedAnimation(
       parent: _controller,
       curve: curve
     );
 
-    return inverted ? new Tween<double>(
+    animation = inverted ? new Tween<double>(
       begin: 1.0,
       end: 0.0
     ).animate(animation) : animation;
+
+    animation.addStatusListener((AnimationStatus status) {
+      setState(() {
+        // This just triggers a rebuild. The state lives in the animation controller.
```

Comment:
Maybe mention in the comment that the reason we want to do this is to update the logic that depends on _isTransitioning in the build method.